Week1/homework/exercise1.js

```diff
@@ -0,0 +1,97 @@
+let mysql = require("mysql");
+
+connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'root',
+    database: 'company',
+});
+
+connection.connect((err) => {
+    if (err) {
+        throw err;
+    }
+    console.log('My sql connected...');
+});
+
+
+connection.query(
+    'CREATE TABLE Employees (emp_no INT PRIMARY KEY, emp_name VARCHAR(50), salary INT, reports_to VARCHAR(255));',
+    function(err, result) {
+        if (err) throw err;
+        console.log('Table Employees created');
+    });
+
+connection.query(
+    'CREATE TABLE Departments (dept_no INT NOT NULL, dept_name VARCHAR(255), manager VARCHAR(50));',
+    function(err, result) {
+        if (err) throw err;
+        console.log('Table Departments created');
+    });
+
+
+connection.query(
+    'CREATE TABLE Projects (proj_no INT PRIMARY KEY, proj_name VARCHAR(255), starting_date DATETIME, ending_date DATETIME );',
+    function(err, result) {
+        if (err) throw err;
+        console.log('Table Projects created');
+    });
+
+const employees = [
```

Comment:
The `emp_no` column could instead be `AUTO_INCREMENT`, so you wouldn't need to supply the IDs.

Reply:
Forgive me, we hadn't learned that in Week 1.